src/lib.rs

```diff
@@ -18,6 +21,10 @@ where
         }
     }
 
+    pub fn destroy(self) -> I2C {
```

Comment:
I think we should put some rustdoc on this function. 

I think the reason this needs to exist is to allow the caller to be able to easily extract the I2C handle for re-use elsewhere? Would the alternative be letting the `Ads1119` be dropped and then re-creating the I2C handle? Is this a pattern you saw somewhere else?